src/main/java/net/neoforged/neoforge/common/extensions/IItemExtension.java

```diff
@@ -63,13 +66,21 @@ private Item self() {
     }
 
     /**
-     * ItemStack sensitive version of getItemAttributeModifiers
+     * ItemStack sensitive version of getDefaultAttributeModifiers. Used when a stack has no {@link DataComponents#ATTRIBUTE_MODIFIERS} component.
      */
     @SuppressWarnings("deprecation")
     default ItemAttributeModifiers getAttributeModifiers(ItemStack stack) {
```

Comment:
Since we're fiddling here, should we rename this to `getDefaultAttributeModifiers` to match the renamed vanilla method?

Or we can just do it in a different PR.